src/Toggle/Toggle.js

```diff
@@ -204,7 +209,7 @@ class Toggle extends React.Component {
     );
 
     if (this.state.switched) {
-      thumbStyles.marginLeft = `-${thumbStyles.width}`;
+      thumbStyles.marginLeft = 0 - thumbStyles.width;
```

Comment:
This is resulting to NaN when thumbStyles width is set to something like '10px'!